### PR TITLE
Announcing the Lunatech Beginner Quarkus Course

### PR DESCRIPTION
**Pull request by njlbenn**
This is part of our marketing campaign around Java and Quarkus. The announcement is styled off the Scala 2 to Scala 3 announcement that we made for a similar campaign: https://blog.lunatech.com/posts/2020-06-08-announcing-the-moving-from-scala-2-to-scala-3-course

- [x] Content review
- [x] Grammar check

Once these two steps are done all that is left is to hit the merge button!

### CODE REVIEW

File: posts/2021-03-22-announcing-the-lunatech-beginner-quarkus-course.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+= Announcing the Lunatech Beginner Quarkus Course
+Neil Benn <neil.benn@lunatech.fr>
```
Comment (pepite) [inline posts/2021-03-22-announcing-the-lunatech-beginner-quarkus-course.adoc:2]: it should be your GitHub handle ;)

Comment (ckipp01) [reply posts/2021-03-22-announcing-the-lunatech-beginner-quarkus-course.adoc:2]: 😆 I keep seeing this comment. I'll add it to the template so it reminds people.

Comment (njlbenn) [reply posts/2021-03-22-announcing-the-lunatech-beginner-quarkus-course.adoc:2]: yea, all I did here was follow the `g8` template instructions, and the example it gave was Nicolas's email address:

```
[neilbenn@MacBook-Pro-de-Neil lunatech-blog (master)]$ g8 file://.
[info] resolving Giter8 0.13.1...

title [The title of you post]: Announcing the Lunatech Beginner Quarkus Course
date [2019-05-01]: 2021-03-22
author [Nicolas Leroux <nicolas@lunatech.com>]: Neil Benn <neil.benn@lunatech.fr>

Template applied in /Users/neilbenn/code/lunatech-blog/.
```

**Review by ckipp01 — APPROVED** [review]
Really cool @njlbenn. LGTM 🚀

**Pull request merged**